views/index.ejs

```diff
@@ -128,6 +128,34 @@
   <br>
   <br>
   
+  <script>
+  document.onkeydown = function(e) {
+    if (e.which == 49) {
+	  alert("Go to Home Page");
```

Comment:
Maybe say "Going to Home Page" instead

Reply:
I was planning on taking those out and making only one alert appear at the beginning explaining navigation. Additionally I am going to add the hotkeys in parenthesis next to the pages so for example Home (1)